vertical-pod-autoscaler/pkg/updater/logic/updater.go

```diff
@@ -155,6 +165,17 @@ func (u *updater) RunOnce() {
 	timer.ObserveTotal()
 }
 
+func getRateLimiter(evictionRateLimit float64, evictionRateLimitBurst int) *rate.Limiter {
+	var evictionRateLimiter *rate.Limiter
+	if evictionRateLimit == -1 || evictionRateLimit == 0 {
+		evictionRateLimiter = rate.NewLimiter(rate.Inf, evictionRateLimitBurst)
+		klog.Warning("Rate limit disabled")
+	} else {
+		evictionRateLimiter = rate.NewLimiter(rate.Every(time.Duration(1.0/evictionRateLimit*float64(time.Second))), evictionRateLimitBurst)
```

Comment:
If I understand the documentation correctly the first parameter is the rate per second, so we could just use `rate.NewLimiter(rate.Limit(evictionRateLimit))`

Reply:
See https://github.com/golang/time/blob/master/rate/rate.go#L25

Reply:
+1 to this comment

Reply:
👍 I missed that thank you @bandesz